cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -854,6 +855,7 @@ void SbPlayerPipeline::CreatePlayer(SbDrmSystem drm_system) {
     } else {
       error_message = player_bridge_->GetPlayerCreationErrorMessage();
       player_bridge_.reset();
+      video_stream_ = nullptr;
```

Comment:
This is not necessary, as |video_stream_| is associated with the |demuxer_|.

The same to the one below.

Reply:
Fixed.